cypress/integration/pages/articles/tests.js

```diff
@@ -122,16 +122,16 @@ export const testsThatFollowSmokeTestConfig = ({
 
               cy.get('figure')
                 .eq(0)
```

Comment:
This test only ever tests the first `figure` in the page. Is that the aim?

Reply:
I think that's deliberate for several reasons, including:
- not wanting to over-test
- the images lower down the page expected to be lazy-loaded
But also I didn't know how to make it work for `.eq(1)` because it was still trying to use the copyrightHolder from the first image.